tests/fixtures/config/account/environment/region/security_groups.yaml

```diff
@@ -5,3 +5,9 @@ parameters:
   param4: {{ environment_path.1 }}
   param5: {{ environment_path.2 }}
   param6: {{ environment_config.region }}
+  param7: {{ stack_config.name }}
+  param8: {{ stack_config.name.upper() }}
+  param9: {{ stack_config.name.0 }}
+  param10: {{ stack_config.name[-2:] }}
+  param11: {{ stack_config.name.split("_")[-1] }}
+  param12: {{ stack_config.name }}-string
```

Comment:
params 8 to 12 are irrelevant as the are testing jinja's functionality not sceptre's.

Reply:
True, but if it's documented functionality, we don't explicitly test for jinja's usage in the config tests. This way if we change rendering engine we don't need to updates the tests, happy to drop the tests if it's easier though

Reply:
Yup - but I feel its safe to document functionality thats been tested and documented in itself. This is testing specifically that rendering framework conforms the jinja templating specification. The onus of testing that should be within the rendering framework itself - if it claims to render jinja. Also v unlikely we'll change from jinja2.